### ISSUE TITLE
Build fails, when Net 5 is set as a Build Tool

### ISSUE THREAD

**Issue by van800** [godotengine/godot#43717]
After installing .Net 5 build would fail with: 
```
  Microsoft.Common.CurrentVersion.targets(1180, 5): [MSB3644] The reference assemblies for .NETFramework,Version=v4.7.2 were not found. To resolve this, install the Developer Pack (SDK/Targeting Pack) for this framework version or retarget your application. You can download .NET Framework Developer Packs at https://aka.ms/msbuild/developerpacks
```

As a workaround I have added [the following](https://github.com/van800/godot-demo-projects/commit/2a2c227c0d9c76a98a4ac9795a93a96f5d2eba63) to my csproj.

I don't know exactly how this can be fixed in godot source. Must be in [here](https://github.com/godotengine/godot/blob/cc3c671f3b636cbfac9fc74095d5bee753e71ed5/modules/mono/editor/Godot.NET.Sdk/Godot.NET.Sdk/Sdk/Sdk.targets#L19-L33).

**Comment by akien-mga** [comment]
See also #43458.

**Comment by neikeq** [comment]
Is this an issue with Godot 3.2 or latest master (4.0-dev)?

**Comment by van800** [comment]
It is the issue with release 3.2.3. I tried on Linux Ubuntu.

**Comment by TomNCatz** [comment]
Reproduced on Windows 10, godot 3.2.3
As a note, this was not an issue in godot 3.2.2 on the same machine

**Comment by neikeq** [comment]
I'm making a PR that will fix this in Godot 3.2.4. For 3.2.3 one can add the following property to their csproj:

```csproj
<GodotUseNETFrameworkRefAssemblies>true</GodotUseNETFrameworkRefAssemblies>
```

**Comment by akien-mga** [comment]
Fixed by #44135.
